.github/ISSUE_TEMPLATE/other.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Other
+about: Any other topic you want to talk about.
+title: ''
+labels: question
+assignees: ''
+---
+
+[comment]: <> (If the Issue happen in Multiple SDK open it in integration-guide instead https://github.com/meilisearch/integration-guides/issues)
```

Comment:
Not sure this remark is useful here

Reply:
You are right. I even wonder if it might not be better to remove the Other. I think we can let the option to create a blank Issue like it's already the case.